app/konnect/dev-portal/service-documentation.md

```diff
@@ -3,82 +3,111 @@ title: Dev Portal Service Documentation
 no_version: true
 ---
 
-Upload documentation for your Services through ServiceHub to display it
-on the Dev Portal.
+A core function of the Developer Portal is uploading Service descriptions and API specs. Developers can use the Spec and corresponding descriptions to access, consume, and register new applications against your Services. For a step by step tutorial on publishing a Service Description and an API Spec to the Developer Portal see our [Get Started: Set Up and Access Dev Portal](/konnect/getting-started/publish-service) guide.
 
-You can upload Service descriptions and version specs. At a minimum, we
-recommend uploading an API spec for each Service version.
-
-After adding documentation for your Services,
-[publish](/konnect/dev-portal/publish) them to the Dev Portal.
 
 ## Service Descriptions
 
-Provide extended descriptions of your Services with Markdown. The
+Service Descriptions Provide extended descriptions of your Services with Markdown. The
 description applies to the whole Service, and appears on every version of that
-Service in the Dev Portal.
+Service in the Dev Portal. The contents of this markdown file will be displayed as the introduction to your API when this Service is published to the Dev Portal.
 
-### Add a Markdown File for a Service
+### Upload a Service Description
 
-1. From the left navigation menu, open the {% konnect_icon servicehub %}
-**ServiceHub**.
+1. From the left navigation menu, open the **ServiceHub** page.
 
 2. Select a Service to open its overview.
 
 3. In the **Service Document** section, click **Upload Document**.
 
-3. Select an `.md` file to upload. Click **Open**.
+3. Select a `.md` file to upload.
+
+### Update a Service Description
 
-### Update or Delete a Markdown File for a Service
+You can upload a new document to replace an existing Service Document
 
-Upload a new document to replace an existing Service document.
+1. From the left navigation menu, open the {% konnect_icon servicehub %} **ServiceHub** page.
 
-1. From the Services page, select a Service to open its overview.
+2. Select a Service to open its overview.
 
-2. In the **Service Document** section:
+3. In the **Service Document** section:
     * To upload a new file, click the **file icon**, then
     **Replace**. Select a new `.md` file to upload.
```

Comment:
Turn this:

> 3. In the **Service Document** section:
    * To upload a new file, click the **file icon**, then
    **Replace**. Select a new `.md` file to upload.

Into this:

> 3. In the **Service Document** section, click the **file icon**, then **Replace**. Select a new `.md` file to upload.

or split into two steps, if you prefer.